server/service/src/sensor/update.rs

```diff
@@ -55,10 +54,6 @@ pub fn validate(
     }
 
     if let Some(location_id) = &input.location_id {
-        if location_id == &("None".to_string()) {
-            // return OK if location_id is None (unassigning this sensor from a location)
-            return Ok(sensor_row);
-        }
```

Comment:
I thought this check was important because otherwise the database returns an error when the 'none' location cannot be found

Reply:
Ah right! (: please add a clearer comment for this in the file. Thanks Ferg

Reply:
nice! Thank you :)